cmd/batch-status.go

```diff
@@ -61,25 +62,41 @@ func mainBatchStatus(ctx *cli.Context) error {
 	defer cancel()
 
 	_, e := client.DescribeBatchJob(ctxt, jobID)
-	fatalIf(probe.NewError(e), "Unable to lookup job status")
+	if madmin.ToErrorResponse(e).Code != "XMinioAdminNoSuchJob" {
```

Comment:
```suggestion
	if e != nil && madmin.ToErrorResponse(e).Code != "XMinioAdminNoSuchJob" {
```
The else block too may require similar changes.